examples/benchmark/scene_test_1.py

```diff
@@ -143,6 +144,7 @@ def on_mouse_move(self, event):
             self.update()
 
     def on_mouse_wheel(self, event):
+        prof = Profiler()  # noqa
```

Comment:
I call `prof('message')` from inside the function to subdivide the measurement. Better to have just one way of doing it?